packages/teuchos/numerics/src/Teuchos_SerialDenseHelpers.hpp

```diff
@@ -55,6 +55,9 @@
 #include "Teuchos_SerialBandDenseMatrix.hpp"
 #include "Teuchos_SerialDenseVector.hpp"
 
+#include "Teuchos_CommHelpers.hpp"
+#include "Teuchos_DefaultComm.hpp"
```

Comment:
Note to self: This is OK, since TeuchosNumerics has a required dependency on TeuchosComm.